svcomp/bench/toInstallDir/data/index.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
Done.  Also, the python3 requirement has been removed.

(I did this because I've just been used to programming in python3...  There were only a few print calls that had to be modified.)